kubedriver/helmclient/client.py

```diff
@@ -69,56 +78,110 @@ def __helm_cmd(self, *args):
         return cmd
 
     def install(self, chart, name, namespace, values=None):
-        args = ['install', chart, '--name', name, '--namespace', namespace]
+        if self.helm_version.startswith("3"):
+            if namespace is not None:
+                args = ['install', name, chart, '--namespace', namespace]
+            else:
+                args = ['install', name, chart]
+        else:
+            if namespace is not None:
+                args = ['install', chart, '--name', name, '--namespace', namespace]
+            else:
+                args = ['install', chart, '--name', name]
         if values != None:
             args.append('-f')
             args.append(values)
         cmd = self.__helm_cmd(*args)
+
         process_result = subprocess.run(cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
-        if process_result.returncode != 0:
+
+        if process_result.returncode == 127:
+            raise CommandError(f'Helm install command not found: {process_result.stdout}')
```

Comment:
Can we rename CommandError to HelmBadCommandError or something like that? It's basically an alternative exception to HelmError, to indicate the command being called is wrong - so it's like a BadRequest error and should be named as such.